app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -0,0 +1,111 @@
+package ar.com.wolox.android.example.ui.login;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.android.example.ui.home.HomeActivity;
+import ar.com.wolox.android.example.ui.signup.SignUpActivity;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+import butterknife.BindView;
+
+import static ar.com.wolox.android.example.utils.Constants.WOLOX_WEB;
+
+/**
+ * Login fragment
+ */
+public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
+
+    @BindView(R.id.vUserEmailInput)
+    EditText vUserEmailInput;
+
+    @BindView(R.id.vUserPasswordInput)
+    EditText vUserPasswordInput;
+
+    @BindView(R.id.vLoginButton)
+    Button vLoginButton;
+
+    @BindView(R.id.vSignUpButton)
+    Button vSignUpButton;
+
+    @BindView(R.id.vLoginTermsAndConditions)
+    TextView vLoginTermsAndConditions;
+
+    @Override
+    public int layout() {
+        return R.layout.fragment_login;
+    }
+
+    @Override
+    public void init() {
+        loadLocalLoginData();
+    }
+
+    @Override
+    public void setListeners() {
+        vLoginButton.setOnClickListener(v -> attemptToLogin());
```

Comment:
I think you can do vLoginButton.setOnClickListener( {attemptToLogin} ); Let me now if you can't archive that.

Reply:
I couldn't change it. I can do it using Kotlin.

Reply:
oh right. My mind was in kotlin there lol.